helm-charts/konk/scripts/provision.sh

```diff
@@ -41,14 +41,14 @@ then
     --from-file=/etc/kubernetes/pki/apiserver-etcd-client.key
   kubectl -n $NAMESPACE label secret $FULLNAME-apiserver-cert $LABELS
 fi
-if ! kubectl -n $NAMESPACE get secret $FULLNAME-ca
+if [ "$(kubectl -n $NAMESPACE get secret $FULLNAME-ca --ignore-not-found 2>&1 )" = "" ]
```

Comment:
We actually need this technique for every place `get secret` is called. The cleanest way to do it would be to write a function.